handbook/engineering/batch-changes/ce-onboarding.md

```diff
@@ -0,0 +1,62 @@
+# Batch Changes CE onboarding
+
+Welcome to Batch Changes! This page is primarily for new members of the CE team, but it could be helpful to other teams, too! After going through these steps, you should:
+
+- understand what Batch Changes is
+- be able to present it to customers from a value, usage, and functional perspective
+- be comfortable demoing Batch Changes to developers and answering questions
+
+## Prerequisites
+General [CE onboarding](../../ce/onboarding.md)
```

Comment:
Do we want to make sure they have the CLI installed ahead of time?

Reply:
This is a step in the quickstart https://docs.sourcegraph.com/batch_changes/quickstart
Do you think that does not suffice?

Reply:
Apologies, forgot it was in there. That's fine!